kerncraft/models/ecm.py

```diff
@@ -228,60 +223,22 @@ def __init__(self, kernel, machine, args=None, parser=None):
                     parser.error('--asm-block can only be "auto", "manual" or an integer')
 
     def analyze(self):
-        # For the IACA/CPU analysis we need to compile and assemble
-        asm_name = self.kernel.compile(
-            self.machine['compiler'], compiler_args=self.machine['compiler flags'])
-        bin_name = self.kernel.assemble(
-            self.machine['compiler'], asm_name, iaca_markers=True, asm_block=self._args.asm_block,
-            asm_increment=self._args.asm_increment)
-
-        # Making sure iaca.sh is available:
-        if find_executable('iaca.sh') is None:
-            print("iaca.sh was not found. Make sure it is found in PATH.", file=sys.stderr)
-            sys.exit(1)
-
         try:
-            cmd = ['iaca.sh', '-64', '-arch', self.machine['micro-architecture'], bin_name]
-            if self._args.verbose >= 3:
-                print('Executing:', ' '.join(cmd))
-            iaca_output = subprocess.check_output(cmd).decode('utf-8')
-        except OSError as e:
-            print("IACA execution failed:", ' '.join(cmd), file=sys.stderr)
-            print(e, file=sys.stderr)
-            sys.exit(1)
-        except subprocess.CalledProcessError as e:
-            print("IACA throughput analysis failed:", e, file=sys.stderr)
+            iacaAnalysis, asm_block = self.kernel.iaca_analysis(self.machine['compiler'],
```

Comment:
Interesting... I was 99% sure that PEP8 says something about local variable naming conventions. Oh well, I like the lower_case_with_underscores style for non-loop-local variables ;)